app/connectors/ApplicationConnector.scala

```diff
@@ -46,46 +46,46 @@ abstract class ApplicationConnector(implicit val ec: ExecutionContext) extends R
 
   def http: HttpClient = if (useProxy) proxiedHttpClient.withHeaders(bearerToken, apiKey) else httpClient
 
-  def updateRateLimitTier(applicationId: String, tier: RateLimitTier)
+  def updateRateLimitTier(applicationId: ApplicationId, tier: RateLimitTier)
                          (implicit hc: HeaderCarrier): Future[ApplicationUpdateResult] = {
-      http.POST[UpdateRateLimitTierRequest, HttpResponse](s"$serviceBaseUrl/application/$applicationId/rate-limit-tier",
+      http.POST[UpdateRateLimitTierRequest, HttpResponse](s"$serviceBaseUrl/application/${applicationId.value}/rate-limit-tier",
```

Comment:
Could we not add a method for base application url that takes the applicationId and gives you back that first 3 sections, rather than repeating this code.